bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -472,40 +432,34 @@ public Object getValueAt(Object node, int columnIndex) {
     }
 
     // Return the world coordinates for the given target
-    private static Option<WorldCoords> getWorldCoords(SPTarget tp, Option<Long> when) {
+    private static Option<WorldCoords> getWorldCoords(final SPTarget tp, final Option<Long> when) {
```

Comment:
@sraaphorst - Correct, the "Distance" in the Target Environment table is the angular distance in arcminutes between the base position and the target.